appuio/redis/values.yaml

```diff
@@ -1006,3 +1006,8 @@ podDisruptionBudget:
   enabled: false
   minAvailable: 1
   # maxUnavailable: 1
+
+## @param forceUpdate Force update the StatefulSet.
+## If enabled the chart will recreate the StatefulSet without touching the Pods (cascade orphan), allowing you to update locked field, such as PVC size.
+##
+forceUpdate: false
```

Comment:
I'm somewhat torn. On one hand we have helm-docs that can parse the values.yaml file for this kind of comments and then put it in the README but it requires a certain comment style. We would have to decide whether to only add this value for now, or make it for the whole document in order to be consistent (applying the boy scout rule here). I would be fine with opening an issue and accept it as technical debt though.
```suggestion
# -- forceUpdate Force update the StatefulSet.
# If enabled the chart will recreate the StatefulSet without touching the Pods (cascade orphan), allowing you to update locked field, such as PVC size.
forceUpdate: false
```